http-api/src/lib.rs

```diff
@@ -420,3 +463,20 @@ fn project_info(urn: Urn, paths: Paths) -> Result<Info, Error> {
         head: head.id.to_string(),
     })
 }
+
+fn get_project_metadata(urn: &Urn, paths: &Paths) -> Result<project::Metadata, Error> {
+    let storage = ReadOnly::open(paths)?;
+    let project = identities::project::get(&storage, urn)?.ok_or(Error::NotFound)?;
+    let meta: project::Metadata = project.try_into()?;
+
+    Ok(meta)
+}
+
+fn get_remote_head_branch(branch_name: &str, peer_id: &PeerId) -> Result<git::Branch, Error> {
```

Comment:
This can be called `remote_branch`, since "head branch" doesn't really mean anything, and "get" implies some kind of fetching/query. Can we also remove the `Result`?